tools/debugger/debugger_tester.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+ESCARGOT=$1
+TEST_CASE=$2
+DEBUGGER_CLIENT=$3
+START_DEBUG_SERVER="${ESCARGOT} --start-debug-server ${TEST_CASE}.js &"
+eval ${START_DEBUG_SERVER}
```

Comment:
I think you can evaluate `${ESCARGOT} --start-debug-server ${TEST_CASE}.js &` directly here, you don't need an 'eval'.